packages/scaffold-ui/src/views/w3m-account-view/index.ts

```diff
@@ -1,18 +1,43 @@
-import { ConnectorController, StorageUtil } from '@reown/appkit-core'
+import {
+  ConnectorController,
+  StorageUtil,
+  type AccountType,
+  AccountController
+} from '@reown/appkit-core'
 import { customElement } from '@reown/appkit-ui'
 import { LitElement, html } from 'lit'
 import { ConstantsUtil as CommonConstantsUtil } from '@reown/appkit-common'
+import { state } from 'lit/decorators.js'
 
 @customElement('w3m-account-view')
 export class W3mAccountView extends LitElement {
-  // -- Render -------------------------------------------- //
+  // -- Members ------------------------------------------- //
+  private unsubscribe: (() => void)[] = []
+
+  // -- State & Properties -------------------------------- //
+  @state() public allAccounts: AccountType[] = AccountController.state.allAccounts
+
+  public constructor() {
+    super()
+    this.unsubscribe.push(
+      AccountController.subscribeKey('allAccounts', val => (this.allAccounts = val))
+    )
+  }
 
+  public override disconnectedCallback() {
+    this.unsubscribe.forEach(unsubscribe => unsubscribe())
+  }
+
+  // -- Render -------------------------------------------- //
   public override render() {
     const connectorId = StorageUtil.getConnectedConnectorId()
     const authConnector = ConnectorController.getAuthConnector()
+    const isAuthConnector = connectorId === CommonConstantsUtil.CONNECTOR_ID.AUTH
+
+    const hasMultipleAccounts = this.allAccounts.length > 1
 
     return html`
-      ${authConnector && connectorId === CommonConstantsUtil.CONNECTOR_ID.AUTH
+      ${hasMultipleAccounts && authConnector && isAuthConnector
```

Comment:
This is not correct, wallet views should be rendered for all embedded wallet networks, what should be conditionally rendered is the settings screen

Basically if user clicks on profile while on a wallet view and they only have a single account, then go directly to settings